apps/api/src/app/shared/services/distributed-lock/distributed-lock.service.ts

```diff
@@ -47,7 +56,7 @@ export class DistributedLockService {
     }
 
     // TODO: Implement distributed nodes (at least 3 Redis instances)
-    this.instances = [getRedisUrl()].filter((instance) => !!instance).map((url) => new Redis(url));
+    this.instances = [getRedisObject()].filter((instance) => !!instance).map((url) => new Redis(url));
```

Comment:
```suggestion
    this.instances = [getRedisObject()].filter((instanceSetings) => !!instanceSettings).map((redisSettings) => new Redis(redisSettings));
```
We also need to change the filter function. A helper method that checks the existence at least of the host and the port (unless `ioredis` defaults properly when passing the settings object.

Reply:
@p-fernandez Correct me if I am wrong, but I believe line 15 covers this.

Reply:
@Cliftonz yes, I missed that. With the suggestion should be good to go.